types/tipset.go

```diff
@@ -119,6 +119,20 @@ func (ts TipSet) MinTicket() (Signature, error) {
 	return ts.blocks[0].Ticket, nil
 }
 
+// MinTimestamp returns the smallest timestamp of all blocks in the tipset.
+func (ts TipSet) MinTimestamp() (Uint64, error) {
+	if len(ts.blocks) == 0 {
+		return 0, errUndefTipSet
+	}
+	min := ts.blocks[0].Timestamp
+	for i := 0; i < len(ts.blocks[0:]); i++ {
```

Comment:
Was this intended to be `len(ts.blocks[1:])`?

Reply:
Whoops, yes thanks!